db/migrate/20230308132125_add_devise_to_users.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[7.0]
+  def self.up
+    change_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
```

Comment:
- [ ] [OPTIONAL] Hi team, you have done really well setting up devise :clap: , I just have one suggestion, please notice that we already have an email column for our user model, so when we run the migration it gives the duplication error like :point_down: 
![image](https://user-images.githubusercontent.com/97521557/223947210-1b248bc5-7935-4703-9394-615aea1377c8.png)

I think what you can do is either comment out this line from the devise generated migration or create a migration to remove the users email column and then run this one :+1: